scripts/runfabtests.sh

```diff
@@ -191,18 +239,19 @@ function unit_test {
 		return
 	fi
 
-	start_time=$(date '+%s.%N')
+	start_time=$(date '+%s')
 
-	${ssh} ${SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
+	${SSH_SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
 	p1=$!
 
 	wait $p1
 	ret1=$?
 
-	end_time=$(date '+%s.%N')
+	end_time=$(date '+%s')
 	test_time=$(compute_duration "$start_time" "$end_time")
 
-	if [ $ret1 -eq 61 ]; then
+        local nr=$(not_run $ret1)
```

Comment:
The "real errors" you're talking about are build errors, right?  That's a good point.  The motivation was to get around the fact that fi_msg_epoll is not built for OSX (no epoll).  The decision to not built it is made at configure time, and it seemed a shame to duplicate that logic in some form in this script.

If that's the only reasonable solution, then I can change it to do that, but I'm open to other ideas.